7_child_vaccination.do

```diff
@@ -1,83 +1,215 @@
-
 ******************************
 *** Child vaccination ********
 ******************************   
-
 *c_measles	child			Child received measles1/MMR1 vaccination
-        gen c_measles  =. 
-		replace c_measles = 1 if (h9 ==1 | h9 ==2 | h9 ==3)  
-     	replace c_measles = 0 if h9 ==0  				
-	
+		gen c_measles  = inlist(h9,1,2,3)   if !inlist(h9,8,9,.)  

```

Comment:
This is not as general as the original code, but if you have reviewed all the surveys and sure that only 8,9 are missing it's fine. However, if there are cases where 99 is coded as missing, this command can not capture it as missing instead would code as 0.

Reply:
Yeah, you are right. I did check all points and found it is ok with VI. But to make sure it applies to all versions, I will change it back.